docs/content/configuration/caching.md

```diff
@@ -32,6 +34,39 @@ A simple in-memory LRU cache. Local cache resides in JVM heap memory, so if you
 |`druid.cache.initialSize`|Initial size of the hashtable backing the cache.|500000|
 |`druid.cache.logEvictionCount`|If non-zero, log cache eviction every `logEvictionCount` items.|0|
 
+### Caffeine Cache
+
+A highly performant local cache implementation for Druid based on [Caffeine](https://github.com/ben-manes/caffeine). Requires a JRE8u60 or higher
```

Comment:
> Requires a JRE8u60 or higher

Does it really, or is that just if you use COMMON_FJP?

Reply:
just for `COMMON_FJP`